synapse/config/registration.py

```diff
@@ -98,6 +73,30 @@ def read_config(self, config, **kwargs):
             config.get("account_validity") or {}, config
         )
 
+        # Load account validity templates.
+        # We do this here instead of in AccountValidityConfig as read_templates
+        # relies on state that hasn't been initialised in AccountValidityConfig
```

Comment:
This is due to the `AccountValidityConfig` not being part of the huge config object that normally gets created? I wonder if there's a good reason for that.

Reply:
Indeed, and my thought is that this class was just created to clean up `RegistrationConfig` a bit, rather than making `AccountValidityConfig` a proper config class.

But I'm not sure if there's any justification against not doing so. May have to wait for people to come back to ask.

Reply:
We talked a bit about this and the conclusion was to attempt to make it a proper config class, but if it turns out to be too hard then to punt it.